lomap/dbmol.py

```diff
@@ -435,12 +447,12 @@ def ecr(mol_i, mol_j):
             # The total score will be the product of all the single rules
                
             tmp_scr = ecr_score * MC.mncar() * MC.mcsr()
-            
             strict_scr = tmp_scr *  MC.tmcsr(strict_flag=True) 
             loose_scr = tmp_scr * MC.tmcsr(strict_flag=False) 
-        
+            #tanimoto = MC.mtansr()
```

Comment:
Why do we have something commented out here? If you don't need it just remove it.